ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -547,9 +548,11 @@ public static string method_toString()
                 "Comparisons|peak_width_base_et\t" + peak_width_base_et.ToString(),
                 "Comparisons|min_peak_count_ee\t" + min_peak_count_ee.ToString(),
                 "Comparisons|min_peak_count_et\t" + min_peak_count_et.ToString(),
-                "Families|family_build_folder_path\t" + family_build_folder_path,
-                "Families|deltaM_edge_display_rounding\t" + deltaM_edge_display_rounding.ToString()
+                //this doesn't seem like it makes sense to be a part of load and run -LVS
```

Comment:
Do we want a file export to be part of an automatic run through though? Doesn't that run the risk of accidentally replacing old files with new ones (I do that at least haha). It makes more sense to me to explicitly/consciously click on something to have it exported... let me know what you think and I can easily uncomment it